writingIdomatic.py

```diff
@@ -17,14 +17,15 @@
 # Notes:    Don’t repeat a '==' comparison in an 'if' statement, use a list or set when you can. ‘x’ in ‘some_list’ can return true or false
 # Ex:
 
+NAME = 'Thomas'
+
 is_generic_name = False
-name = 'Thomas'
-if name == 'Thomas' or name == 'Amir' or name == 'Brett':
+if NAME == 'Thomas' or NAME == 'Amir' or NAME == 'Brett':
     is_generic_name = True
 print(is_generic_name) # prints 'True'
 
-name = 'Thomas'
-is_generic_name = name in ('Thomas', 'Amir', 'Brett')
+is_generic_name = False
+is_generic_name = NAME in ('Thomas', 'Amir', 'Brett')
```

Comment:
is_generic_name = name in (THOMAS, AMIR, BRETT)

Reply:
Updated